### DISCUSSION TITLE
Devices show unavailable home assistant

### DISCUSSION THREAD

**Discussion by margag02** [Koenkk/zigbee2mqtt#10105] [Devices]
Hi, all my zigbee devices connected to zigbee2mqtt in home asistant show as unavailable every 15 seconds

**Comment by margag02** [comment]
i forgot to mention i use conbeeII

**Comment by Dutchy-79** [comment]
Same problem here using a sonoff zigbee 3.0 dongle plus

Everything used in zigbee2mqtt is unavailable on the same time. Then it's available again etc.

When manually turning off a device with the physical zigbee switch the light stays on in HA and Z2M.

Therefore I think it's a Z2M issue because it does not update the new state even after 30min
